### PR TITLE
feat: adding V2 functions workflow for load tests PAGOPA-855

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Addition of scripts in the load test section to test the V2 soap functions

#### Motivation and Context
This addition was required to properly test the newly implemented functions and their capabilities, confronted with the previous ones.

#### How Has This Been Tested?
This has been tested through k6 and devops pipelines in environment DEV

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FedericoRuzzier** [comment]
update_code

**Comment by FedericoRuzzier** [comment]
update_code

File: performance-test/run_performance_test.sh [hunk 1]
```diff
@@ -8,7 +8,8 @@ APICONFIG_SUBSCRIPTION_KEY=$5
 GPD_SUBSCRIPTION_KEY=$6
 GPS_SUBSCRIPTION_KEY=$7
 DONATIONS_SUBSCRIPTION_KEY=$8
-DEBT_POSITION_NUMBER=$9
+SOAP_SUBSCRIPTION_KEY=$9
```
Comment (AngeloCaporaso) [inline performance-test/run_performance_test.sh:11]: Maybe you could consider using a single variable for all micro-services as in the integration tests

File: performance-test/src/test-types/load.json [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
   "stages": [
     {
-      "duration": "2m",
-      "target": 100
+      "duration": "5s",
+      "target": 10
     },
     {
-      "duration": "1m",
+      "duration": "0m",
```
Comment (AngeloCaporaso) [inline performance-test/src/test-types/load.json:18]: I guess `duration = 0 ` it's a typo related to some test or debug

**Review by AngeloCaporaso — APPROVED** [review]
Good job 🥇 

Consider whether to apply the suggested changes.

**Comment by AngeloCaporaso** [comment]
If possible, please indicate in the PR description what environment they were tested in

**Pull request merged**